src/dotnet-interactive/Commands/Quit.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.App.Commands
+{
+    public class Quit : KernelCommandBase
+    {
+
+        public Quit(Action onQuit = null, string targetKernelName = null): base(targetKernelName)
+        {
+            onQuit ??= DefaultOnQuit ??(() =>
+            {
+                Environment.Exit(0);
```

Comment:
The second fallback is in case `DefaultOnQuit` is `null`. Why allow `DefaultOnQuit` to be `null`?

Reply:
makes sense, setting it to null thrown